src/routes/list-paginated.js

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+import listBuckets from '../storage/bucket/list.js';
+import { listObjectsPaginated } from '../storage/object/list.js';
+import { getChildRules, hasPermission } from '../utils/auth.js';
+
+export default async function getListPaginated({ req, env, daCtx }) {
+  if (!daCtx.org) return listBuckets(env, daCtx);
```

Comment:
Should we even support listing buckets on this new API endpoint? With the infrastructure move, this shouldn't be needed anymore, as we have all sites in one bucket @auniverseaway .